src/hooks/useWalletConnection.ts

```diff
@@ -6,7 +6,7 @@ import { BlockchainUpdatePrompt, WalletInfo } from 'api/wallet/WalletApi'
 
 interface PendingStateObject extends WalletInfo {
   pending: true
-  networkIdOrDefault: number
+  networkIdOrDefault: Network
```

Comment:
is this true @W3stside @Velenir ?

```js
export enum Network {
  Mainnet = 1,
  Ropsten = 3,
  Rinkeby = 4,
  Goerli = 5,
  Kovan = 42,
  xDAI = 100,
}
```

Bein network a enum, we assume that this value won't ever be any other network. 
Or the logic is setting mainnet if we are in an Unknown network?